README.md

```diff
@@ -44,54 +62,126 @@ This goes both ways: when explaining something make sure to present enough conte
+![Review States on a ticket](./assets/reviewStates.png). 
+6. **Rejected**: Reviewer (product manager or tech lead) has reviewed the work to address ticket and rejected it.
+7. **Accepted**: Reviewer (product manager or tech lead) has reviewed the work to address ticket and accepted it.
+
+##### ⚙️ Chore Ticket
+1. **Planned**: Ready for current iteration.
+2. **Unstarted**: No work has begun to address the ticket.
+3. **Started**: Work has begun to address the ticket.
+4. **Accepted**: Work to address ticket is completed, pull request was submitted, reviewed and merged.
+
+### Code Review and Pull Requests
+
+We prefer code that's more readable and easier to understand over code that's concise or obfuscated. We strive to use the most understandable variable names in their respective context, and even favor longer, easier to understand variable names over abbreviated ones.
+
+We also prefer DRY code, and favor small, single purpose functions over long functions that try to do too much.
+
+We use these [general guidelines](https://github.com/DoSomething/communal-docs/tree/master/Contributing) to guide how we contribute to our codebases.
+
+We work in branches, generally off of dev (or master, depending on the project). We commit often, to capture our thinking. We package up our work as pull requests, and try to keep PRs reasonably small. (This also makes it more likely someone will review your PR!)
+
+We push changes multiple times a day to (remote) branches. We always push changes at the end of the day.
 
 A teammate must review your PR, get their questions answered, and give a +1 as approval before you can merge your changes. The main branch is [protected](https://help.github.com/articles/about-required-reviews-for-pull-requests/). The reason we review code is not just to catch bugs, but because we want to improve the quality of code that we ship. We want our code to be maintainable, easily understood, and [stylish](https://github.com/DoSomething/code-style). If you push code to production, you’re implicitly vouching for it as an example of how to write code at DoSomething. When possible, we use StyleCI to enforce coding style and conventions. StyleCI gets called for each PR and update.
 
-#### Making a pull request
+#### Making a Pull Request
 
-* If your code is not ready for review, or is a prototype that won't be merged, don't create a pull request—or create one prefixed with DON’T MERGE in the title
-* When you create a pull request, fill out the [pull request template](https://github.com/blog/2111-issue-and-pull-request-templates)
-* Highlight any areas you would particularly like review on
+* If your code is not ready for a full review, create a [draft pull request](https://github.blog/2019-02-14-introducing-draft-pull-requests/). These are great for getting initial feedback on a direction you're taking. If it's a prototype that won't be merged, create a pull request prefixed with DON’T MERGE in the title. 
```

Comment:
Worth calling out specifically any best practices around pushing up draft PRs if you're blocked to facilitate discussion and/or any kind of advice over when it's been too long and you should unblock yourself by pushing up the PR sooner v later? For example, if you're working on something, is 1 days full heads down a good time to give yourself before you step back and ask yourself if asking others for help would be valuable?

Reply:
I think it varies by level of experience and size of work. Because for a JSE, we would expect a timebox of < 2 hours

Reply:
I think that could be a valuable benchmark for especially new people to the team to see!